readthedocs/rtd_tests/tests/test_resolver.py

```diff
@@ -374,14 +370,23 @@ def test_domain_resolver_with_domain_object(self):
             url = resolve_domain(project=self.pip)
             self.assertEqual(url, 'docs.foobar.com')
 
-    @override_settings(PRODUCTION_DOMAIN='readthedocs.org')
+            url = resolve_domain(project=self.pip, use_custom_domain=False)
+            self.assertEqual(url, "pip.readthedocs.io")
```

Comment:
this is probably the main test, since the other ones should return the same response.